pkg/controller/kedacontroller/kedacontroller_controller.go

```diff
@@ -322,15 +319,13 @@ func (r *ReconcileKedaController) installMetricsServer(instance *kedav1alpha1.Ke
 	if len(instance.Spec.LogLevelMetrics) > 0 {
 		transforms = append(transforms, transform.ReplaceMetricsServerLogLevel(instance.Spec.LogLevelMetrics, r.scheme, log))
 	}
-	if err := r.resourcesMetrics.Transform(transforms...); err != nil {
+	manifest, err := r.resourcesMetrics.Transform(transforms...)
+	if err != nil {
 		log.Error(err, "Unable to transform Metrics Server manifest")
 		return err
 	}
+	r.resourcesMetrics = manifest
 
-	if err := r.resourcesMetrics.ApplyAll(); err != nil {
```

Comment:
some kind of `Apply()` method is no longer needed? Maybe I am missing something, but what triggers the creation of resources after they are transformed?